CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+## 1.49.12
```

Comment:
I think this should bump the minor version, since it's effectively a new feature.